package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.1.1",
   "dependencies": {
     "gm": "~1.21.1",
-    "parse": "^1.3.5"
+    "parse": "^3.0"
   },
```

Comment:
This should be 2.0 as 3.0 is not released in the JS SDK yet

Reply:
of course. done.